README.md

```diff
@@ -44,13 +47,18 @@ waiting for the tests to run.
 
 This command produces `assemble/target/accumulo-<version>-bin.tar.gz`
 
+## Contributing
+
+Contributions are welcome to all Apache Accumulo repositories
+
+If you want to contribute, go through our guide [How to contribute](https://accumulo.apache.org/how-to-contribute/)
```

Comment:
```suggestion
Contributions are welcome to all Apache Accumulo repositories.

If you want to contribute, read [our guide on our website][contribute].
```

Can put `[contribute]: https://accumulo.apache.org/how-to-contribute` at the end.